packages-openshift.yaml

```diff
@@ -189,6 +189,18 @@ postprocess:
     ---
     EOF
 
+  # Inject symlink for Two Node OpenShift with Fencing (TNF)
+  # This is a workaround to allow pacemaker to use a pre-release resource agent for etcd
+  # This should be removed after https://issues.redhat.com/browse/OCPEDGE-1706 is completed
+  # We touch an empty file in the target location to pass kola tests
+  - |
+    #!/usr/bin/env bash
+    set -xeuo pipefail
+    mkdir -p /usr/lib/ocf/resource.d/ocp-tnf
+    ln -s /usr/local/lib/ocf/resource.d/ocp-tnf/podman-etcd /usr/lib/ocf/resource.d/ocp-tnf/podman-etcd
+    mkdir -p /var/usrlocal/lib/ocf/resource.d/ocp-tnf
+    touch /usr/local/lib/ocf/resource.d/ocp-tnf/podman-etcd
```

Comment:
I don't think you need those. They will be created on demand when you install the files there.

To workaround the broken symlink kola check, you can skip by adding an entry in https://github.com/openshift/os/blob/master/kola-denylist.yaml instead.

This should check if the package is fixed and fail if it is.

Reply:
With regards to ensuring the symlink is not present once the package is fixed, I spoke to the team about this and our concern is that this would break our installations (and upgrades) until we patch CEO. We intentionally installed this symlink at a non-conflicting location from the one that will be used when the package is fixed. Once the underlying fix is in the RPM, we need to update the our code in CEO to ensure that upgrades and installs all reference the new version. Otherwise, we risk having the code point to where the symlink no longer is.

The advantage of doing it this way is that we can ensure that upgrades always pair a compatible CEO and RHCOS.

Reply:
If that's really unacceptable to the team, can land this alternative for now and work on a bug fix for CEO.
https://github.com/openshift/os/pull/1796